README.md

```diff
@@ -1,2 +1,51 @@
 # skupper-docker
 A command-line tool for setting up and managing Skupper docker installations
+
+Note: If you are using Fedora 32, use the https://fedoramagazine.org/docker-and-fedora-32/ procedure to install Docker.
+
+To create the `skupper-docker` command:
+
+1. Clone this repo.
+2. Run the following command in the repo root directory:
+```
+ make build-cmd
+```
+After this command is completed, you can create a skupper site:
+```
+ ./skupper-docker init
+```
+
+To troubleshoot issues:
+
+1. Run `docker ps` to view the containers:
+
+```
+$ docker ps
+
+CONTAINER ID        IMAGE                                       COMMAND                  CREATED             STATUS              PORTS                                                 NAMES
+3c3b93460544        quay.io/skupper/skupper-docker-controller   "/app/controller"        50 minutes ago      Up 50 minutes                                                             skupper-service-controller
+d8c9edc2f4a0        quay.io/interconnectedcloud/qdrouterd       "/home/qdrouterd/bin…"   50 minutes ago      Up 50 minutes       5671-5672/tcp, 9090/tcp, 45671/tcp, 55671-55672/tcp   skupper-router
+```
+
+2. Log into the router container using the CONTAINER ID, for example:
+
+```
+$ docker exec -it d8c9edc2f4a0 /bin/bash
```

Comment:
Rather than use the container id, it might be easier to use the container name =>

docker exec -it skupper-router /bin/bash